html_css/v2/main-course/grid-lessons/positioning-grid-items.md

```diff
@@ -0,0 +1,161 @@
+
+### Cells
+
+Cells are the smallest unit of measurement in a grid. You can think of grid cells like a cell in a spreadsheet or a square on a graph. By default each child element of a grid container will occupy one cell. In the example above, we have 9 cells in our grid. So the element we marked with the letter "A" is a cell between row grid lines 1 and 2 and column grid lines 1 and 2. The element with the letter "H" is a cell between row grid lines 3 and 4 and column grid lines 2 and 3.
+
+But what happens if we wanted to change the order of our grid items? Or we want items to occupy more than one cell?
+
+### Positioning
+
+To get a feel for how items can be positioned we will create a mock floor plan for an apartment. Let's start with a total area of our apartment (the grid container) divided into a 5x5 grid. To make this example a little clearer, we'll use a background color to distinguish our container space. Note that we're also using `display: inline-grid` here so that our container does not stretch to take up space the way a block-level box would. This will just help us better visualize the space.
+
+<p class="codepen" data-height="300" data-default-tab="css,result" data-slug-hash="qBXyqvp" data-editable="true" data-user="dm-murphy" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span>See the Pen <a href="https://codepen.io/dm-murphy/pen/qBXyqvp">
+  Second-Positioning-Loft</a> by Daniel (<a href="https://codepen.io/dm-murphy">@dm-murphy</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+
+As it stands this is a pretty sad unit. To make it less of an empty box and more of a home we are going to add some items to our grid container that will represent different rooms.
+
+<p class="codepen" data-height="300" data-default-tab="css,result" data-slug-hash="XWaBpje" data-editable="true" data-user="dm-murphy" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span>See the Pen <a href="https://codepen.io/dm-murphy/pen/XWaBpje">
+  Third-Positioning-Rooms</a> by Daniel (<a href="https://codepen.io/dm-murphy">@dm-murphy</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+
+Most of our rooms represent a single grid cell. But we have given ourselves a large living room (yay!) We positioned this item using `grid-column-start` and `grid-column-end`. Their property values represent the column grid lines we wish it to start and end with. 
```

Comment:
Should be a period after (yay!) to complete the sentence

Reply:
Sorry, I split the sentences up here and forgot to fix this. Because it's an exclamatory sentence inside the parentheses the "yay" will need to be capitalized and I'll just end the prior sentence at "living room."